k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -16,13 +16,26 @@ function usage() {
   echo "                                     default file path is ${solution_yaml}."
   echo "    -n|--nodename NODENAME: collects logs from pods running only on given node".
   echo "                            collects logs from all the nodes by default."
+  echo "    --duration DURATION : dutaion for which logs should be collected"
+  echo "    --size_limit SIZE : max log size "
+  echo "    --binlogs True/False : option to collect binary logs"
+  echo "    --coredumps True/False : option to collect core dumps"
+  echo "    --stacktrace True/False : option to collect stack trace"
+  echo "    --all True/False : if True, sets --binlogs --coredumps & --stacktrace to True"
```

Comment:
aggregated option to collect binlogs, coredumps & stacktrace.
If set to true, It overrides --binlogs, --coredumps & --stacktrace at once.

Reply:
done